sdk/communication/azure-communication-identity/tests/recordings/test_communication_identity_client.pyTestClienttest_get_token_from_managed_identity[chat&voip-value2].json

```diff
@@ -0,0 +1,246 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0/.well-known/openid-configuration",
```

Comment:
is `72f988bf-86f1-41af-91ab-2d7cd011db47` sensitive information?

Reply:
BTW I can access it in in-private mode without any authz so it should be safe to indicate. If it was a sensitive resource, it would have some auth flow at least